src/main/java/duke/commands/LocationSearchCommand.java

```diff
@@ -0,0 +1,36 @@
+package duke.commands;
+
+import duke.requests.LocationSearchUrlReq;
+import duke.commons.DukeException;
+import duke.storage.Storage;
+import duke.ui.Ui;
+import duke.api.ApiParser;
+
+import java.io.IOException;
+import java.util.ArrayList;
+
+import com.google.gson.JsonObject;
+
+/**
+ * Class representing a command to send the test URL connection.
+ */
+public class LocationSearchCommand extends Command {
+    private String param;
+
+    public LocationSearchCommand(String param) {
+        this.param = param;
+    }
+
+    /**
+     * Executes this command with given param.
+     */
+    @Override
+    public void execute(Ui ui, Storage storage) throws IOException, DukeException {
+        ArrayList<String> result = ApiParser.getLocationSearch(param);
+
```

Comment:
Write new method for showSearch and moved the String constant "These are the coordinates ...." into Ui.